site/content/en/docs/Reference/fleet.md

```diff
@@ -83,6 +83,27 @@ spec:
         logLevel: Info
         grpcPort: 9357
         httpPort: 9358
+      #
+      # [Stage:Alpha]
+      # [FeatureFlag:CountsAndLists]
+      # Counts and Lists provides the configuration for generic (player, room, session, etc.) tracking features.
+      # Commented out since Alpha, and disabled by default
+      # counters: # counters are int64 counters stored against a GameServer in the fleet that can be incremented and decremented by set amounts. Keys must be declared at Fleet creation time.
+      #   games: # arbitrary key.
+      #     count: 0 # initial value.
+      #     capacity: 100 # (Optional) Defaults to 1000 and setting capacity to max(int64) may lead to issues and is not recommended. See GitHub issue https://github.com/googleforgames/agones/issues/3636 for more details.
```

Comment:
```suggestion
      #     capacity: 100
```

Thanks for adding the Counter and List example here - was missing!

Since this is the GameServer template, I think we can keep this pretty low on the description, since that's what the GameServer reference is for -- I'd bring this down to the level we have in https://github.com/googleforgames/agones/blob/release-1.40.0/examples/fleet.yaml

Some for the other fields.

Reply:
Changed in 188f49126f57d8a50f322431e6e249a6cd141e9f and made both files more consistent.